tasties_app/models.py

```diff
@@ -46,6 +55,37 @@ class UnitChoices(models.TextChoices, Enum):
     )
     description = models.CharField(max_length=64, validators=[MinLengthValidator(1)])
 
+    def __str__(self):
+        return str(self.amount) + " " + self.measurement_unit + " " + self.description
+
+    @staticmethod
+    def is_enum_value(value):
+        """
+        Checks if the given value is a valid UnitChoices enum value.
+        """
+        for unit in Ingredient.UnitChoices:
+            if unit == value:
+                return True
+        return False
+
+    @staticmethod
+    def edit_info(ingredient, amount, measurement_unit, description):
```

Comment:
Why not giving default values so if  we will want to edit just one of the fields we will not have to pass all the arguments?